creator-node/src/services/stateMachineManager/stateReconciliation/issueSyncRequest.jobProcessor.js

```diff
@@ -95,8 +114,38 @@ module.exports = async function ({ logger, syncType, syncRequestParameters }) {
     return {
       error: {
         message: errorMsg
+      },
+      jobsToEnqueue: {}
+    }
+  }
+
+  if (syncMode === SYNC_MODES.MergePrimaryAndSecondary) {
+    if (!mergePrimaryAndSecondaryEnabled) {
+      logger.info(
+        `${logMsgString} || Sync mode is disabled - Will not issue sync request`
+      )
+      return {
+        error: {},
+        jobsToEnqueue: {}
```

Comment:
same thing here -- if this returns early, do you think it's worth logging a metric to get visibility into when this sync mode is disabled? I'm leaning toward "no" but I guess it could be good to know if some SPs have it disabled (seems unlikely tho)

Reply:
responded above